crates/top/re_sdk/tests/flush.rs

```diff
@@ -0,0 +1,22 @@
+use std::time::Duration;
+
+use re_sdk::RecordingStreamBuilder;
+
+#[test]
+fn test_drop_grpc_sink() {
+    re_log::setup_logging();
+    let url_to_nowhere = "rerun+http://not.real:1234/proxy";
+
+    re_log::info!("Connecting…");
+    let rec = RecordingStreamBuilder::new("rerun_example_grpc_drop_test")
+        .connect_grpc_opts(url_to_nowhere)
```

Comment:
this is a great example for needing to configure connection time out - as is, this is a quite long running test